README.md

```diff
@@ -52,6 +52,8 @@ Content
 * **server_extensions_api** - contains the extension API that all extensions use to extend CAS' functionality.
 * **server_extensions_extra** - reference extension implementation that demonstrates BTC, LTC, CLOAK, DGB, DASH, HATCH, POT, VIA, BTX, SYS, FLASH, DOGE, NLG, ICG, NBT, GRS, MAX, BSD, MEC, BTDX, NANO, SUM, BURST, ECA, LINDA, $PAC, DAI, MKR, BTBS, MUE, BAT and REP coin support functionality.
 * **server_extensions_test** - contains tester for testing the extensions (CAS not required).
+* **operators_sample_website** -- <a href="https://github.com/GENERALBYTESCOM/batm_public/tree/master/operators_sample_website">The OSW</a> is our new WebApp written in PHP and based on the Laravel framework and its RestServices.
```

Comment:
Remove "new" from the sentence - it may be new now but won't be forever. Also please change this description - it should provide information about the purpose of the application